test/standalone/ez80_builtin/src/main.c

```diff
@@ -121,7 +121,7 @@ static int test_ctzc(void) {
     unsigned char input;
     for (int i = 0; i < 256; i++) {
         input = (unsigned char)i;
-        truth = (input == 0) ? 8 : __builtin_ctz((unsigned int)input);
+        truth = (input == 0) ? 8 : __builtin_ctz((unsigned int)input | INT_MIN);
```

Comment:
What is the purpose of this change?

Reply:
Pretty much just making sure the optimizer is confused enough to avoid narrowing the CTZ operation and thus doing a tautology test (I saw this happen for 48-bit, though not 8-bit at this point)